django/utils/ipv6.py

```diff
@@ -10,139 +7,32 @@
 def clean_ipv6_address(ip_str, unpack_ipv4=False,
                        error_message=_("This is not a valid IPv6 address.")):
     """
-    Cleans an IPv6 address string.
+    Clean an IPv6 address string.
 
-    Validity is checked by calling is_valid_ipv6_address() - if an
-    invalid address is passed, ValidationError is raised.
+    If an invalid address is passed, ValidationError is raised.
 
-    Replaces the longest continuous zero-sequence with "::" and
-    removes leading zeroes and makes sure all hextets are lowercase.
+    Replace the longest continuous zero-sequence with "::", remove leading
```

Comment:
I guess the `ipaddress` module does these things? I'm not sure if it's behavior should be reiterated in detail.